src/main/java/net/minecraftforge/client/CloudRenderer.java

```diff
@@ -0,0 +1,495 @@
+                            buffer.pos(slice, 0,      sectZ0).tex(sliceCoord0, v0).color(0.9F, 0.9F, 0.9F, ALPHA).endVertex();
+                            buffer.pos(slice, HEIGHT, sectZ0).tex(sliceCoord1, v0).color(0.9F, 0.9F, 0.9F, ALPHA).endVertex();
+                            buffer.pos(slice, HEIGHT, sectZ1).tex(sliceCoord1, v1).color(0.9F, 0.9F, 0.9F, ALPHA).endVertex();
+                            buffer.pos(slice, 0,      sectZ1).tex(sliceCoord0, v1).color(0.9F, 0.9F, 0.9F, ALPHA).endVertex();
+                            slice += INSET;
+                        }
+                    }
+
+                    for (slice = sectZ0; slice < sectZ1;)
+                    {
+                        sliceCoord0 = slice * sectPx;
+                        sliceCoord1 = sliceCoord0 + PX_SIZE;
+
+                        // Z sides
+                        if (slice > -CULL_DIST)
+                        {
+                            slice += INSET;
+                            buffer.pos(sectX0, 0,      slice).tex(u0, sliceCoord0).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            buffer.pos(sectX0, HEIGHT, slice).tex(u0, sliceCoord1).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            buffer.pos(sectX1, HEIGHT, slice).tex(u1, sliceCoord1).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            buffer.pos(sectX1, 0,      slice).tex(u1, sliceCoord0).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            slice -= INSET;
+                        }
+
+                        slice += scale;
+
+                        if (slice <= CULL_DIST)
+                        {
+                            slice -= INSET;
+                            buffer.pos(sectX1, 0,      slice).tex(u1, sliceCoord0).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            buffer.pos(sectX1, HEIGHT, slice).tex(u1, sliceCoord1).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            buffer.pos(sectX0, HEIGHT, slice).tex(u0, sliceCoord1).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            buffer.pos(sectX0, 0,      slice).tex(u0, sliceCoord0).color(0.8F, 0.8F, 0.8F, ALPHA).endVertex();
+                            slice += INSET;
+                        }
+                    }
+                }
+
+                sectZ0 = sectZ1;
+            }
+
+            sectX0 = sectX1;
+        }
+    }
+
+    private void dispose()
+    {
+        if (vbo != null)
+        {
+            vbo.deleteGlBuffers();
+            vbo = null;
+        }
+        if (displayList >= 0)
+        {
+            GLAllocation.deleteDisplayLists(displayList);
+            displayList = -1;
+        }
+    }
+
+    private void build()
+    {
+        Tessellator tess = Tessellator.getInstance();
+        BufferBuilder buffer = tess.getBuffer();
+
+        if (OpenGlHelper.useVbo())
+            vbo = new net.minecraft.client.renderer.vertex.VertexBuffer(FORMAT);
```

Comment:
You don't need the fully-qualified name here now that this is targeting 1.12 - might as well take advantage of the name change.